e2e/case4.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
nit, I usually go with the standard
```
#!/usr/bin/env bash
set -o errexit -o nounset -o pipefail
```

`#!/usr/bin/env bash` is more portable a good habit to use by default

Reply:
Updated bash script.